packages/analytics-client-common/src/attribution/campaign-parser.ts

```diff
@@ -26,6 +26,8 @@ import {
 export class CampaignParser implements ICampaignParser {
   utmCookieStorage = new UTMCookie();
 
+  constructor(protected queryStringParsingOnly: boolean = false) {}
```

Comment:
Wasn't sure what to do here, since utm's stored in cookies is essentially behavior. Other possibilities I considered:
1) Add an optional options object to CampaignParser which includes queryStringParsingOnly, and could be expanded in the future.
2) Copy CampaignParser into the page tracking plugin and remove the cookie code.

Since CampaignParser isn't really a documented part of our SDK, I figured changes like this are less sensitive, but open to other solutions.

Reply:
Should we just remove utmz?